source/beamline/BioXAS/BioXASBeamlineComponent.h

```diff
@@ -13,6 +13,16 @@ class BioXASBeamlineComponent : public AMControl
 	/// Destructor.
 	virtual ~BioXASBeamlineComponent();
 
+	/// Returns true if this control should be able to be stopped (if connected), false otherwise.
+	virtual bool shouldStop() const { return true; }
```

Comment:
Is this true for all BioXASBeamlineComponent classes?  Are components all motors?